spec/models/document_type_selection/option_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "json"
+
+RSpec.describe DocumentTypeSelection::Option do
+  describe ".managed_elsewhere?" do
```

Comment:
As this is an instance method it should be described with `#managed_elsewhere?` - see http://www.betterspecs.org/#describe